### PR TITLE
Re-enabling thermostat preset mode support

### PR DESCRIPTION
**Pull request by akudaikon**
Adding this pull request to close the loop on thermostat presets. This goes along with my other pull request in the api: https://github.com/JoshuaMulliken/wyzeapy/pull/59

### CODE REVIEW

File: custom_components/wyzeapi/climate.py [hunk 1]
```diff
@@ -142,11 +143,17 @@ def target_temperature_low(self) -> Optional[float]:
 
     @property
     def preset_mode(self) -> Optional[str]:
-        raise NotImplementedError
+        # pylint: disable=R1705
+        if self._thermostat.preset == Preset.HOME:
```
Comment (JoshuaMulliken) [inline custom_components/wyzeapi/climate.py:147]: Can we clean this up by doing a switch statement?

**Review by JoshuaMulliken — CHANGES_REQUESTED** [review]
Overall looks good to me! Just need the CLA to clear and have one nitpick about the preset mode property

File: custom_components/wyzeapi/climate.py (hunk 1 above)
Comment (akudaikon) [reply custom_components/wyzeapi/climate.py:147]: Definitely can. I was just reverting back the changes that initially disabled presets in https://github.com/JoshuaMulliken/ha-wyzeapi/issues/207

Comment (JoshuaMulliken) [reply custom_components/wyzeapi/climate.py:147]: Ahh gotcha.

**Review by JoshuaMulliken — APPROVED** [review]
LGTM!

**Pull request merged**